conf/ssh_regenconf_hook

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+force=${2:-0}  # 0/1 --force argument
+dryrun=${3:-0}  # 0/1 --dry-run argument
+pending_conf=$4 # Path of the pending conf file
+
+# https://github.com/YunoHost/yunohost/blob/c7eaeca6486e6240c832b8863791f8b03520a0f0/data/hooks/conf_regen/43-dnsmasq
+
+do_pre_regen() {
+    if [ $dryrun -eq 0 ]
+    then
+        # Remove added lines into the sshd config
+        sed -i "/##-> __APP__/,/##<- __APP__/d" /etc/ssh/sshd_config
+    fi
```

Comment:
I don't understand why you would do this ...
- I don't see any reason to have a custom behavior if dry_run is true
- The pre_regen should *NOT* change the actual configuration files ever. Otherwise they will appear as manually modified which means no more auto-upgrade of these files
- Instead, the actual file to patch should be the one in `${pending_dir}`. See [here](https://github.com/YunoHost-Apps/sympa_ynh/blob/master/sources/hooks/conf_regen/98-postfix_sympa#L9) for an example

Reply:
There's only one good reason to not make a proper regenhook conf for an app.
There's no fucking documentation on this thing, so we have to reverse engineering the whole thing.
Which end up with the first thing apparently working...

Don't hesitate to fix it.